CONTRIBUTING.md

```diff
@@ -105,3 +105,17 @@ To increase coverage if necessary, [write tests using `pytest`].
 For more info on running tests, please refer to ["Running tests"](https://significant-gravitas.github.io/Auto-GPT/testing/).
 
 [write tests using `pytest`]: https://realpython.com/pytest-python-testing/
+
+ 
+In Pytest, we use VCRpy. It's a package that allows us to save OpenAI and other API providers' responses.
+When you run Pytest locally:
+- If no prompt change: you will not consume API tokens because there are no new OpenAI calls required.
```

Comment:
Broken list: https://docs-git-fork-merwanehamadi-feature-integration-f300b2-auto-gpt.vercel.app/Auto-GPT/contributing/#thread-id=Jvb0N
```suggestion

- If no prompt change: you will not consume API tokens because there are no new OpenAI calls required.
```